### PR TITLE
Added Event Cards Animation

### LINKED ISSUE #282: feat: Adding Loader and Animation to the event cards
**Is your feature request related to a problem? Please describe.**
When i am changing the filters then the cards will be directly removed or shown which doesn't looks good i want to add a 1s loader so that it will create an effect like it is loading the cards and also on changing the filters i want to add the coming animation on the cards .

**Describe the solution you'd like**
I want to add the loader and adding animations after user change the filter  to the cards

https://github.com/Tanay-ErrorCode/lupo-skill/assets/118350936/c9ac0c34-694f-4598-b7b4-a34b87a49009

### PR DESCRIPTION
**Pull request by Ayushmaanagarwal1211**
**What kind of change does this PR introduce?**
This pr include the change in the animation of the cards .
When cards comes in user viewport then it will shown with an animation of fade-up and when out of port then it will become invisible .
Sir if you want any changes please let me know 

**Issue Number:**

Fixes #282 

**Snapshots/Videos:**

https://github.com/Tanay-ErrorCode/lupo-skill/assets/118350936/d8675be9-e7d3-4677-8919-1d5f6c810311

**Other information**

I used aos library to this 
Sir if you found it suitable for level 2 then please give level 2

### CODE REVIEW

**Comment by Ayushmaanagarwal1211** [comment]
@Tanay-ErrorCode  please review it

**Comment by Ayushmaanagarwal1211** [comment]
@Tanay-ErrorCode  Hey! Can you please check

File: aos.d.ts [hunk 1]
```diff
@@ -0,0 +1,20 @@
+declare module 'aos' {
```
Comment (Doraemon012) [inline aos.d.ts:1]: Why have you added this file? Please explain.

Comment (Ayushmaanagarwal1211) [reply aos.d.ts:1]: @Doraemon012  To configure it for aos library

Comment (Ayushmaanagarwal1211) [reply aos.d.ts:1]: Aos is used for adding aniamtion to it

**Comment by Tanay-ErrorCode** [comment]
@Ayushmaanagarwal1211 Any updates>

**Comment by Ayushmaanagarwal1211** [comment]
Yes i am trying to do this day after tomorrow my exams is being finished so i will surely do that after that

**Comment by Tanay-ErrorCode** [comment]
@Ayushmaanagarwal1211 Are you still working on this issue?

**Pull request closed without merging**